client/main.lua

```diff
@@ -550,7 +550,7 @@ end)
 
 AddEventHandler('onResourceStart', function(resource)
     if GetCurrentResourceName() == resource then
-        playerJob = exports.qbx_core:GetPlayerData().job
+        playerJob = QBX.PlayerData.job
```

Comment:
QBX.PlayerData will likely not exist when the resource starts since the player hasn't logged into a character yet